models/articles-models.js

```diff
@@ -8,4 +8,26 @@ exports.selectArticleById = (article_id) => {
             return Promise.reject({status: 404, msg: "article does not exist"});
         }
     })
+};
+
+exports.selectAllArticles = () => {
+    return db.query(`SELECT 
+    articles.author, 
+    title, 
+    articles.article_id, 
+    topic, 
+    articles.created_at, 
+    articles.votes, 
+    article_img_url,
+    COUNT(comment_id) AS comment_count 
```

Comment:
I think if you add ::INT after the closing bracket on line 22 - you can avoid having to do the coercing in the next step :)

Reply:
Ah thank you :) I'll get that sorted now and add in the error status to my messages